lib/StaticNarrative/exporter/objects_with_sets.py

```diff
@@ -70,18 +77,28 @@ def _list_objects_with_sets(
 
         processed_refs = {}
         data = []
-        set_ret = self.set_api_client.call_method(
-            "list_sets",
-            [
+
+        if self.set_api_client:
```

Comment:
Leaving this in for this PR only so that the output using the SetAPI client vs doing it all in-house can be tested.